endpoints-framework/src/main/resources/scopeDescriptions.properties

```diff
@@ -0,0 +1,219 @@
+#Source: https://developers.google.com/identity/protocols/googlescopes
```

Comment:
Keys and values were extracted separately using the xpath expressions below, and merged in a spreadsheet.
Then some transformations were performed to deduplicate, sort the entries and add comments.

All of this could be easily automated, but I was hoping you had an "official" list of scope descriptions at Google.
This would be much better than scrapping a web page. Do you know if something like that exists?